lib/Software/License/OpenLDAP_2_7.pm

```diff
@@ -0,0 +1,62 @@
+use strict;
+use warnings;
+package Software::License::OpenLDAP_2_7;
+
+use parent 'Software::License';
+# ABSTRACT: OpenLDAP Public License v2.7
+
+sub name { 'The OpenLDAP Public License Version 2.7, 7 September 2001' }
+sub url  { 'http://www.openldap.org/doc/admin21/license.html' }
+sub meta_name  { 'openldap' }
+sub meta2_name { 'openldap_2_7' }
```

Comment:
Hi @haarg 

Thanks for your suggestions. I have pushed the changes.
Kindly review it again.

Many Thanks.
Best Regards,
Mohammad S Anwar

Reply:
The OpenLDAP license is obviously based on the 3-clause BSD license, but seems sufficiently different that I'm not sure it could be said to be OSI approved.

Reply:
The SPDX website lists it as OSI approved, but I can't find any proof of that on the OSI website. Who would have guessed licenses are a mess…